bwapi/include/BWAPI/UnitType.h

```diff
@@ -908,6 +908,11 @@ namespace BWAPI
     /// @see TechType::whatResearches, UpgradeType::whatUpgrades
     /// @m_since{4,2,0}
     bool isSuccessorOf(UnitType type) const;
+
+    /// <summary>Retrieves this unit type's upgrade type for upgrading speed</summary>
+    ///
+    /// @returns UpgradeType used as this unit type's ground weapon.
+    UpgradeType speedUpgrade() const;

```

Comment:
include 
```
/// @m_since{5,0,0}
```
which will generate a "Added in BWAPI 5.0.0" note in the documentation and automatically link to the section in the changelog with the notes.

Also small nit: period at end of summary.